test/chi/custom-elements/progress.pug

```diff
@@ -0,0 +1,13 @@
+---
+title: Progress
+scripts:
+  - js/ce/ux-chi-ce
+---
+.test-states
+  each state, index in ['default', 'danger', 'info', 'warning', 'success']
+    .example(style='width:1200px')
+      .-p--3
+        h3.-mb--3= `Progress / ${state}`
+        each value, index in [0,25,50,75,100]
+          h5= `${value}%`
+          chi-progress(state=`${state === 'default' ? '' : state}`, value=value, data-cy=`progress-${value}-${state}`)
```

Comment:
@srinath-koneru Please, add test with different max value in both, custom element and normal version.

Reply:
@dani-cl-madrid copy.